pymatgen/io/cp2k/inputs.py

```diff
@@ -2173,7 +2173,7 @@ def pairwise(iterable):
         return cls(kpoint_sets=kpoint_sets, filename="BAND.bs")
 
 
-@deprecated(BandStructure, "Deprecated on 2024-03-29.", deadline=(2025, 3, 29))
```

Comment:
I removed the date here as I don't think user would care about the date it was marked as deprecated, but the data it would be removed

Reply:
as a developer, i definitely care when it was deprecated. of course that information can be retrieved from `git` history but is less convenient. it could be a code comment rather than part of the message but i'd prefer to keep a record of how long a deprecation period was decided

Reply:
Okay, let revert it then :)